apps/xen-tg-app/src/components/Settings.tsx

```diff
@@ -36,7 +36,7 @@ const Settings = () => {
     queryKey: ['memories'],
     queryFn: queryFn('getMemories', {
       offset: 0,
-      limit: 10,
+      limit: 1,
```

Comment:
Why did you choose to make the size of the page so small?

Reply:
Because I need only the "total" parameter here

Reply:
😆 why not 0?

Reply:
Done